requirements.txt

```diff
@@ -1 +1,2 @@
-git+https://github.com/illinois-cs241/chainlink@0.0.2
+git+https://github.com/illinois-cs241/chainlink@0.0.3
+websockets
```

Comment:
Do we need `jsonschema` here too?

Reply:
I think we do since the source seems to be using it.

Reply:
fixed!